### PR TITLE
Fix bug where discussion page-load would overwrite grade with first rubric

### PR DESCRIPTION
**Pull request by JamieTran**

### CODE REVIEW

File: components/right-panel/consistent-evaluation-rubric.js [hunk 1]
```diff
@@ -171,7 +171,7 @@ class ConsistentEvaluationRubric extends LocalizeConsistentEvaluation(RtlMixin(L
 			this.rubricWindowPopout.close();
 		}
 
-		if (changedProperties.has('rubricInfos')) {
+		if (changedProperties.has('rubricInfos') && this.activityType !== 'discussionActivity') {
```
Comment (maddiegabriel) [inline components/right-panel/consistent-evaluation-rubric.js:174]: should we use the `discussionActivity` constant like in consistent-evaluation-page?
https://github.com/BrightspaceHypermediaComponents/consistent-evaluation/blob/bb5752023a28a29fa205d40261cf8c108cbf1875/components/consistent-evaluation-page.js#L544

**Review by maddiegabriel — APPROVED** [review]
LGTM

File: components/right-panel/consistent-evaluation-rubric.js (hunk 1 above)
Comment (Scy-D2L) [reply components/right-panel/consistent-evaluation-rubric.js:174]: Typically I'd agree but I think the right-panel checking on the activity type is meant to be a temporary thing until we get the rubrics list enabled in discussions, so adding additional wiring for that might not be necessary 🤷🏼

**Review by Scy-D2L — APPROVED** [review]
lgtm

**Pull request merged**